src/containers/quiz/quiz-header/QuizHeader.tsx

```diff
@@ -17,6 +17,9 @@ type QuizHeaderProps = {
   questionsAnswered?: number
   createdAt: string
   updatedAt: string
+  isTimeLimit?: boolean
```

Comment:
You probably mean that the quiz has a time limit here.
```suggestion
  hasTimeLimit?: boolean
```

Reply:
Fixed.